validator.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+func main() {
+	var conf, doc string
+	flag.StringVar(&doc, "doc", "file:///examples/invalid/invalid_email.json", "Example file")
+	flag.Parse()
+	schemaLoader := gojsonschema.NewReferenceLoader("file:///schema.json")
```

Comment:
We might work around using a file. Either by providing the json directly or embeding the file into go code. Erfan is doing something similar for reposeed.

Reply:
@stp yes ack!

Reply:
@stp maybe I did not get you here but I feel like now we are providing the json directly as this "file:///" notation is as per the xepuvv vendor package neither it is a flag value anymore so why we need to move using a file?

Reply:
`file:///schema.json` still makes assumptions on how the code is run. It hardcodes the path of the schema so that the loader can only run inside the schema/ directory.
We can go with the current idea for now, but it's a bug in my opinion to hardcode such assumptions.
A few questions that need answering:
* When the code is run?
* When is it actually runable without errors?
* Could we have the schema.json added as golang struct or via packr so we don't rely on it being on the filesystem?
* Do we assume a development setup for this code to be runable? If yes, do we have documentation ready to explain how to set this up and run this code?
Did we test the various edgecases, when is this code run etc.